src/stringhomework/StringDemo.java

```diff
@@ -0,0 +1,188 @@
+        return String.valueOf(input.charAt(index));
+    }
+
+    // Method For Task2
+    public int calculateSequenceQuantity(String input) {
+        int count = 0;
+        char[] chars = new char[input.length()];
+
+        for (int i = 0; i < input.length(); i++) {
+            chars[i] = input.charAt(i);
+            if (((chars[0] != ' ') && (i == 0)) || ((i > 0) && (chars[i] != ' ') && (chars[i - 1] == ' ')))
+                count++;
+        }
+
+        return count;
+    }
+
+    // Method For Task3
+    public String getReverse(String input) {
+
+        if (input == null || input.isEmpty()) {
+            return Constants.NO_SYMBOL;
+        }
+
+        StringBuilder stringBuilder = new StringBuilder(input);
+
+        if (stringBuilder.toString().equals(stringBuilder.reverse().toString())) {
+            return "Palindrome";
+        } else return "Not a palindrome";
+
+    }
+
+    // Method For Task4
+    public String checkParenthesis(String input) {
+        int counter = 0;
+        boolean flag;
+
+        if (input == null || input.isEmpty()) {
+            return Constants.NO_SYMBOL;
+        }
+
+        for (int i = 0; i < input.length(); i++) {
+
+            if (input.charAt(i) != '(' && input.charAt(i) != ')') {
+                return Constants.STRANGE_SYMBOL;
+            }
+
+            if (input.charAt(i) == '(') {
+                counter++;
+            } else if (input.charAt(i) == ')') {
+                counter--;
+            }
+        }
+
+        flag = counter == 0;
+
+        if (flag) {
+            return "Valid";
+        } else return "Invalid";
+    }
+
+    // Method For Task5
+    public String checkContainSequence(String input, String sequence) {
+        if (input == null || input.isEmpty()) {
+            return Constants.NO_SYMBOL;
+        }
+
+        if (sequence == null || sequence.isEmpty()) {
+            return Constants.NO_SYMBOL;
+        }
+
+        return String.valueOf(input.contains(sequence));
+    }
+
+    // Method For Task6
+    public String checkStartsWith(String input, String value) {
+        if (input == null || input.isEmpty()) {
+            return Constants.NO_SYMBOL;
+        }
+
+        if (value == null || value.isEmpty()) {
+            return Constants.NO_SYMBOL;
+        }
+
+        return String.valueOf(input.startsWith(value));
+    }
+
+    // Method For Task7
+    public String checkEndsWith(String input, String value) {
+        if (input == null || input.isEmpty()) {
+            return Constants.NO_SYMBOL;
+        }
+
+        if (value == null || value.isEmpty()) {
+            return Constants.NO_SYMBOL;
+        }
+
+        return String.valueOf(input.endsWith(value));
+    }
+
+    // Method For Task8
+    public String removeDuplicatedChars(String input) {
+        if (input == null || input.isEmpty()) {
+            return Constants.NO_SYMBOL;
+        }
+        return Arrays.stream(input.split("")).distinct().collect(Collectors.joining());
```

Comment:
wow :D

senc el kara lini, motavor senc mi ban).

        char[] arrayNew = input.toCharArray();
        String noDublicates="";
        for (char value:arrayNew){
            if (noDublicates.indexOf(value)==-1){
                noDublicates+=value;
            }
        }